src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -12,20 +18,54 @@
 
 	private static final int SCROLL_TIMEOUT = 100;
 	private static final boolean USE_DEVICE_PIXEL_RATIO = true;
+	private static final int RETINA_SCALE_FACTOR = 2;
 
 	private ScreenshotProvider() {
 		// private constructor for util class
 	}
 
 	public static BufferedImage shootFullPage( final WebDriver driver ) {
+		final BufferedImage image;
 		if ( driver instanceof ChromeDriver ) {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.WHOLE_PAGE_CHROME, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
+			if ( isRetinaDisplay() ) {
+				return resize( image, image.getWidth() / RETINA_SCALE_FACTOR, image.getHeight() / RETINA_SCALE_FACTOR );
+			}
 		} else {
-			return Shutterbug
+			image = Shutterbug
 					.shootPage( driver, ScrollStrategy.BOTH_DIRECTIONS, SCROLL_TIMEOUT, USE_DEVICE_PIXEL_RATIO )
 					.getImage();
 		}
+		return image;
+	}
+
+	// TODO Remove with retest-model version 5.1.0 and use ImageUtils instead.
+	private static BufferedImage resize( final BufferedImage image, final int width, final int height ) {
+		final Image tmp = image.getScaledInstance( width, height, Image.SCALE_SMOOTH );
+		final BufferedImage resized = new BufferedImage( width, height, BufferedImage.TYPE_INT_ARGB );
+		final Graphics2D graphics2D = resized.createGraphics();
+		graphics2D.drawImage( tmp, 0, 0, null );
+		graphics2D.dispose();
+		return resized;
+	}
+
+	public static boolean isRetinaDisplay() {
```

Comment:
As mentioned, this code belongs to retest-model. So add it there, and put a comment here so this method can be removed, once the change "trickles" here.